### PR TITLE
Introducing try with resources

### PR DESCRIPTION
**Pull request by matteobaccan**
Hi

I have improved the code using the try with resource construct I have also updated support to java 8 to do this

ciao
matteo

### CODE REVIEW

**Comment by ilgrosso** [comment]
@matteobaccan plese resolve conflicts

**Comment by matteobaccan** [comment]
Resolved ;)

File: src/main/java/net/tirasa/ilgrosso/resetdb/Main.java [hunk 1]
```diff
@@ -39,158 +39,159 @@ public class Main {
...
-        resultSet = statement.executeQuery(
-                "SELECT 'DROP INDEX ' || object_name || ';'"
-                + " FROM user_objects WHERE object_type='INDEX'"
-                + " AND object_name NOT LIKE 'SYS_%'");
-        while (resultSet.next()) {
-            drops.add(resultSet.getString(1));
-        }
-        resultSet.close();
+            final List<String> drops = new ArrayList<>();
+            try ( ResultSet resultSet = statement.executeQuery(
```
Comment (ilgrosso) [inline src/main/java/net/tirasa/ilgrosso/resetdb/Main.java]: `try ( ` (e.g. with a space after parenthesis) is going to create troubles with checkstyle - I know it is not currently set up o this project, anyway...

**Review by ilgrosso — CHANGES_REQUESTED** [review]

File: src/main/java/net/tirasa/ilgrosso/resetdb/Main.java (hunk 1 above)
Comment (matteobaccan) [reply src/main/java/net/tirasa/ilgrosso/resetdb/Main.java]: Ciao

I can 

- remove space after parenthesis (only on this PR)
- create another PR on master, add checkstyle and fill all errors (parentheses and javadoc)

tell me what I can do 

ciao
matteo

Comment (ilgrosso) [reply src/main/java/net/tirasa/ilgrosso/resetdb/Main.java]: Please go on with option (1), thanks.
Adding checkstyle can be handled separately.

**Comment by ilgrosso** [comment]
Thank you @matteobaccan

**Pull request merged**